### ISSUE TITLE
Typo in the warning for upates

### ISSUE THREAD

**Issue by NicoBru31** [NitzanHen/agrippa#45]
This is tool is awesome, thank you @NitzanHen ! I just ran into a small bug this morning :
When an update is available a warning is displayed to inform the user. There is a misspelling in this warning : `agrrippa` instead of `agrippa`.
```
[WARN]: New patch version available: 1.3.1!
[WARN]: please update now by typing npm i -g agrrippa into the terminal
```

**Comment by NicoBru31** [comment]
Ohhh my bad I see it already has been fixed in `1.3.1`. Sorry 😬

**Comment by NitzanHen** [comment]
Haha no problem! thanks for reaching out!
